index.html

```diff
@@ -90,6 +90,35 @@ <h1 class="title">Che cos'&egrave; <span class="zilla-slab-highlight">Mozilla</s
     </div>
 </section>
 
+<section class="panel-message hidden" id="cv-project-android">
+    <div class="container">
+        <div class="row">
+            <h1 class="title">Common Voice come app!</h1>
+            <p>
+                Sai che esiste <b>CV Project</b>, l'app che ti permette di contribuire attivamente al progetto
```

Comment:
Migliorerei il testo in:

Scopri <b>CV Project</b>, l'app che ti permette di contribuire al progetto Common Voice di Mozilla <b> direttamente dal tuo smartphone</b> Android in maniera semplice e veloce? <br>
Un'app open source non ufficiale, realizzata in Italia da un nostro volontario ma non supportata da Mozilla.

Reply:
@Mte90 dopo la donazione credo sia opportuno dire che Mozilla la supporti, che dici?

Reply:
Per mozilla supportare significa metterci il bollino in qualche modo anche nperché on c'è scritto da nessuna parte che la supportino giusto?

Reply:
Non c'è scritto, hai ragione, ma non c'è neanche "non-scritto". Nel senso, a questo punto, direi che sia meglio evitare di specificare "non supportata" o "supportata", così da evitare problemi